apidoc/Titanium/Media/AudioPlayer.yml

```diff
@@ -372,6 +450,44 @@ properties:
     permission: read-only
     platforms: [iphone, ipad]
 
+  - name: muted
+    summary: Indicates whether or not audio output of the player is muted. 
+    description: Only affects audio muting for the player instance and not for the device.
+    platforms: [iphone, ipad, android]
+    type: Boolean
+    since: "7.4.0"
+    
+  - name: externalPlaybackActive
+    summary: Indicates whether the player is currently playing video in "external playback" mode.
+    platforms: [iphone, ipad]
+    type: Boolean
+    since: "7.4.0"    
+    permission: read-only
+ 
+  - name: allowsExternalPlayback
+    summary: Indicates whether the player allows switching to "external playback" mode.
+    default: true
+    type: Boolean
+    since: "7.4.0"    
+    platforms: [iphone, ipad]
+    
+  - name: rate
+    summary: |
+        Indicates the desired rate of playback; 0.0 means "paused", 1.0 indicates a 
+        desire to play at the natural rate of the current item.
+    description: |
+        Setting the value of rate to 0.0 pauses playback, causing the value of 
+        the `state` property change to <Titanium.Media.AUDIO_STATE_PAUSED>.
+        Setting the rate to a non-zero value causes the value of the `state` property 
+        to become either <Titanium.Media.AUDIO_STATE_WAITING_FOR_QUEUE> or 
+        <Titanium.Media.AUDIO_STATE_PLAYING>, depending on whether sufficient 
+        media data has been buffered for playback to occur and whether the player's 
+        default behavior of waiting in order to minimize stalling is permitted.
+    default: 0.0
```

Comment:
Shouldn't the default rate be 1.0?

Reply:
Also, I would assume setting rate to 2.0 would cause it to play audio twice as fast, right?
We might want to state that to make it extra clear what it does.

Reply:
By default, the rate is 0.0, indicating that it is **not playing**. Once started, it turns to 1.0. Once paused or stopped, it moves to 1.0 again. For x.0, it would play x as fast **or** slow, yep! I will add that to the docs.

Reply:
So, this doesn't quite match my expectation based on my experience with other audio libraries such as OpenAL and Microsoft's XAudio2. In those libraries, rate is a separate setting and would not be 0.0 if paused.

In XAudio2, the equivalent of rate would be `IXAudio2SourceVoice.SetFrequencyRatio()`.
In OpenAL, the equivalent of rate would be `AL_PITCH`.
(I've never used Android's OpenSL library. So, I'm not sure what it is there.)

Is 0.0 what you're getting from Apple?